urls.py

```diff
@@ -21,6 +21,6 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'^cfp($|/)', ProposalCreate.as_view(success_url='/'), name="cfp"),
```

Comment:
I think the ending slash problem can be solved with: 
 http://stackoverflow.com/questions/1596552/django-urls-without-a-trailing-slash-do-not-redirect